ruby/Rakefile

```diff
@@ -95,7 +97,18 @@ else
     ]
   end
 
+  task 'gem:java' do
```

Comment:
ulta nit: should this be jruby to be more clear?

Reply:
I was inclined to align with the terminology used in the [`Gemfile`](https://github.com/protocolbuffers/protobuf/blame/master/ruby/google-protobuf.gemspec#L14) but I don't feel strongly about it.

Reply:
sure thing, was just thinking about future travelers that are reading this, what does "java" mean vs jruby is a clearer target, at least for me.

Reply:
@haberman / @acozzette - any preference between `gem:java` vs `gem:jruby`?

Reply:
Either one sounds OK to me.